### PR TITLE
Add basic tests

### PR DESCRIPTION
**Pull request by jolheiser**
I plan on adding more as I go, but how does this feel for an initial implementation?

Note I've required an env var to run the tests, as they hit github's API and therefore I didn't want them to run _every_ time tests are ran. 

Eventually tests can/should be added that mimic the API so we can test without a full integration.

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -60,6 +56,9 @@ func Init() {
 		color.Magenta("Using local recipe directory: %s", color.HiBlackString(recipeDir))
 		WebmanRecipeDir = recipeDir
 	}
+	if err := os.MkdirAll(WebmanRecipeDir, os.ModePerm); err != nil {
```
Comment (jolheiser) [inline utils/utils.go:59]: The tests already helped catch this bug, where the recipes dir wasn't made ahead of time. 
This means currently new installs of webman are broken.

File: cmd/remove/remove_test.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	assert.NoErr(err) // Command should execute
+
+	_, err = os.Stat(filepath.Join(utils.WebmanBinDir, "jq"))
+	assert.NoErr(err) // jq binary should exist
+
+	err = RemoveCmd.Execute()
+	assert.NoErr(err) // Command should execute
+
+	_, err = os.Stat(filepath.Join(utils.WebmanBinDir, "jq"))
+	assert.True(err != nil) // jq binary should no longer exist
```
Comment (candrewlee14) [inline cmd/remove/remove_test.go:35]: Should probably use `os.IsNotExist(err)` here instead of `!= nil`

Comment (candrewlee14) [inline cmd/remove/remove_test.go:35]: Should also probably check whether `jq` has been deleted from  `WebmanPkgDir`

File: utils/utils.go (hunk 1 above)
Comment (candrewlee14) [reply utils/utils.go:59]: Great catch, and nice job getting these started. There's plenty of areas for testing that probably will catch more.

**Review by candrewlee14 — COMMENTED** [review]
Love it! Just a couple comments, but great work!

File: cmd/remove/remove_test.go (hunk 2 above)
Comment (jolheiser) [reply cmd/remove/remove_test.go]: Done

Comment (jolheiser) [reply cmd/remove/remove_test.go]: Done

**Pull request merged**